src/components/Atoms/AddTaskButton/index.stories.jsx

```diff
@@ -0,0 +1,9 @@
+import { AddTaskButton } from "./index";
+
+export default { component: AddTaskButton };
+
+export const Default = {
+  args: {
+    onclick: () => console.log("clicked"),
```

Comment:
```suggestion
    onClick: () => console.log("clicked"),
```

細かいところですが、現状Storybookでクリックしてもログが表示されないため修正をお願いします。